src/plugins/interactive_setup/public/format_bash_command.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export function formatBashCommand(command: string, args?: string) {
```

Comment:
__optional nit:__ It's not a big deal at all, just wanted to note: it sounds like too much credit to `Bash` shell here ;) Maybe something like `formatCommadLineSnippet\formatCli...` would be more OS/shell agnostic.